templates/rundeck-config.erb

```diff
@@ -31,6 +31,9 @@ dataSource {
   password = "<%= @database_config['password'] %>"
   dialect = "<%= @database_config['dialect'] %>"
   <%- end -%>
+  properties {
+    maxActive = 100
```

Comment:
If the default is 50 and is not enough, 100 might also not be enough in some cases and a waste of resources in another.  Making this tunable is the way to go then: keep the default value by default, but allow to change it if needed.